scipy/stats/tests/test_discrete_distns.py

```diff
@@ -426,7 +432,11 @@ def variance(N, m1, n, w):
             b = (n-u)*(u + m2 - n)
             return N*a*b / ((N-1) * (m1*b + m2*a))
 
-        assert_allclose(nchypergeom_wallenius.stats(N, m1, n, w, moments='v'),
+        with warnings.catch_warnings():
+            warnings.filterwarnings('ignore',
+                    category=RuntimeWarning,
+                    message="invalid value encountered in mean")
+            assert_allclose(nchypergeom_wallenius.stats(N, m1, n, w, moments='v'),
                         variance(N, m1, n, w), rtol=5e-2)
```

Comment:
```suggestion
            shapes = N, m1, n, w
            assert_allclose(nchypergeom_wallenius.stats(*shapes, moments='v'),
                            variance(N, m1, n, w), rtol=5e-2)
```
`scipy/stats/tests/test_discrete_distns.py:439:80: E501 line too long (82 > 79 characters)`

Reply:
This kind of change seems a little excessive, taking an extra line and making things less readable - it's better to just ignore the linter imho.

Reply:
Can't please everyone.